dev-libs/libnl/files/libnl-3.11-nl_cache_resync_v2.patch

```diff
@@ -0,0 +1,80 @@
+diff --git a/include/netlink/cache.h b/include/netlink/cache.h
```

Comment:
Again, this is unlikely to be acceptable. Adding or modifying API is not something we can or should backport.